salt/states/user.py

```diff
@@ -274,6 +275,10 @@ def present(name,
             Additionally, parent directories will *not* be created. The parent
             directory for ``home`` must already exist.
 
+    nologinit : False
+        If set to ``True``, it will not add the user to lastlog and faillog
+        databases.
+
```

Comment:
Can you make the docstring explicit about which platforms support this option? This will help future users of this feature. You can do it like so:

```
    nologinit : False
         If set to ``True``, it will not add the user to lastlog and faillog
         databases.

        .. note::
            This is only supported on <platform goes here>.
```

Thanks!